.github/scripts/checkLiquidation.mjs

```diff
@@ -0,0 +1,157 @@
+// @ts-check
+import fetch from 'node-fetch';
+import assert from 'assert';
+const delay = (ms) => {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+};
+
+const states = {
+  liquidated: 'liquidated',
+  liquidating: 'liquidating',
+};
+
+const expectedLiquidating = {
+  ids: [
+    'published.vaultFactory.managers.manager0.vaults.vault10-liquidating',
+    'published.vaultFactory.managers.manager0.vaults.vault11-liquidating',
+    'published.vaultFactory.managers.manager0.vaults.vault12-liquidating',
+  ],
+  debts: ['100500000', '103515000', '105525000'],
+  balance: ['15000000', '15000000', '15000000'],
+  denom: 'ATOM',
+};
+
+const expectedLiquidated = {
+  ids: [
+    'published.vaultFactory.managers.manager0.vaults.vault10-liquidated',
+    'published.vaultFactory.managers.manager0.vaults.vault11-liquidated',
+    'published.vaultFactory.managers.manager0.vaults.vault12-liquidated',
+  ],
+  debts: ['0', '0', '0'],
+  balance: ['3425146', '3077900', '2846403'],
+  denom: 'ATOM',
+};
+
+const validate = async ({
+  apiUrl,
+  maxRetries,
+  retryDuration,
+  expectedIds,
+  expectedDebts,
+  expectedBalance,
+  expectedDenom,
```

Comment:
these parameters aren't really independent of each other. `expectedLiquidating` and `expectedLiquidated` have the same structure. please just pass in that object.